src/main/java/fi/otavanopisto/kuntaapi/server/persistence/dao/IdentifierDAO.java

```diff
@@ -92,38 +94,38 @@ public Identifier findByTypeSourceAndKuntaApiId(String type, String source, Stri
     return getSingleResult(entityManager.createQuery(criteria));
   }
 
-  /**
-   * Lists identifiers by type and source
-   * 
-   * @param type identifier type
-   * @param source source
-   * @param firstResult first result. Null is interpret as 0
-   * @param maxResults maximum number of results. Specifying null returns all identifiers
-   * @return
-   */
-  public List<Identifier> listByTypeAndSource(String type, String source, Integer firstResult, Integer maxResults) {
-    EntityManager entityManager = getEntityManager();
-
-    CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
-    CriteriaQuery<Identifier> criteria = criteriaBuilder.createQuery(Identifier.class);
-    Root<Identifier> root = criteria.from(Identifier.class);
-    criteria.select(root);
-    criteria.where(
-      criteriaBuilder.and(
-        criteriaBuilder.equal(root.get(Identifier_.type), type),
-        criteriaBuilder.equal(root.get(Identifier_.source), source)     
-      )
-    );
-    
-    TypedQuery<Identifier> query = entityManager.createQuery(criteria);
-    if (firstResult != null) {
-      query.setFirstResult(firstResult);
-    }
-    
-    if (maxResults != null) {
-      query.setMaxResults(maxResults);
-    }
-    
-    return query.getResultList();
-  }
+//  /**
+//   * Lists identifiers by type and source
+//   * 
+//   * @param type identifier type
+//   * @param source source
+//   * @param firstResult first result. Null is interpret as 0
+//   * @param maxResults maximum number of results. Specifying null returns all identifiers
+//   * @return
+//   */
+//  public List<Identifier> listByTypeAndSource(String type, String source, Integer firstResult, Integer maxResults) {
+//    EntityManager entityManager = getEntityManager();
+//
+//    CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+//    CriteriaQuery<Identifier> criteria = criteriaBuilder.createQuery(Identifier.class);
+//    Root<Identifier> root = criteria.from(Identifier.class);
+//    criteria.select(root);
+//    criteria.where(
+//      criteriaBuilder.and(
+//        criteriaBuilder.equal(root.get(Identifier_.type), type),
+//        criteriaBuilder.equal(root.get(Identifier_.source), source)     
+//      )
+//    );
+//    
+//    TypedQuery<Identifier> query = entityManager.createQuery(criteria);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)